openff/evaluator/protocols/reweighting.py

```diff
@@ -491,7 +491,7 @@ def _reweight_observables(
             observable_dimensions = observable.value.shape[1]
             assert observable_dimensions == 1
 
-            results = mbar.computeExpectations(
+            results = mbar.compute_expectations(
```

Comment:
Perhaps, we can include both pymbar v3 and v4 API by using a try-except block.

Reply:
Yup - "my" trick for handling this is copying what I saw Pascal do a few months ago https://github.com/shirtsgroup/physical_validation/pull/220